src/SDKs/ProvisioningServices/Management.ProvisioningServices/Microsoft.Azure.Management.ProvisioningServices.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+	
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ProvisioningServices</PackageId>
```

Comment:
@montgomp any reason your service name in the swagger and your nuget package is not aligned.
For e.g. your nuget is called provisioning services, but your description uses Device Provisioning service and your API version in swagger is called just "devices"
This is just way too many names, your nuget name should be aligned with your service name.

Reply:
I can change to Microsoft.Azure.Management.DeviceProvisioningService if that is the only thing holding up the PR. I cannot change the Microsoft.Devices in the swagger as that is not owned by my team. Please advise if this is acceptable as it is quite a large rework.

Reply:
The swagger changes you suggest would be a breaking change for python CLI that is already released.

Reply:
The reason I have asked this is because, there is way for developers to search for a particular API version in nuget.org
So if your nuget is called ProvisioningService but your actual service is Devices, how will the developer know what to search for?
What is the official name of your service?

We cannot have different names for the service, it has to be consistent across.

Reply:
The service name is "[Microsoft Azure IoT Hub Device Provisioning Service](https://docs.microsoft.com/en-us/azure/iot-dps/)". I think that is quite the marketing mouthful which is why I'm more inclined to namespace as Microsoft.Azure.Management.DeviceProvisioningService. 

Again, I cannot change the Microsoft.Devices references due to 1) not being a contributor to the various projects if you want to go down that rabbit hole. 2) The number of production breaks that would be caused to ARM, portal, CLI, and existing PowerShell cmdlets for IoT Hub and Device Provisioning Service. 

Are you suggesting I move this assembly away from the Microsoft.Azure.Management.XXX hierarchy that other service SDKs appear to use?